cdap-watchdog/src/test/java/co/cask/cdap/logging/write/LogCleanupTest.java

```diff
@@ -261,7 +299,7 @@ public void testDeleteEmptyDir1() throws Exception {
   @Test
   public void testDeleteEmptyDir2() throws Exception {
     // Create base dir
-    LocationFactory locationFactory = injector.getInstance(LocationFactory.class);
+    LocationFactory locationFactory = injector.getInstance(LocalLocationFactory.class);
```

Comment:
Why change this LocationFactory got binded the the instance of LocalLocationFactory created above no ?